### ISSUE TITLE
Trésorerie / Comptes banque

### ISSUE THREAD

**Issue by webaaz** [afup/back-office#25]
iso

**Comment by thierrymarianne** [comment]
Idem que pour les informations serveurs pour les rôles trésorier ? Est-ce bien de cela dont il s'agit ?

**Comment by webaaz** [comment]
iso par rapport au back office actuel, installe le pour voir.

**Comment by thierrymarianne** [comment]
Si c'est iso, on le conserve tel quel peut être ? (Si c'est faisable ping @mikaelkael )

**Comment by webaaz** [comment]
Le code est difficilement adaptable enfin y'aura sûrement du code à
reprendre.

**Comment by thierrymarianne** [comment]
Considères-tu que cet aspect est de plus haute priorité ou moins prioritaire que la gestion des utilisateurs (au sens des inscriptions, désinscriptions) ?

**Comment by webaaz** [comment]
Non, mais ça doit suivre avec la gestion des events et des factures :)

**Comment by thierrymarianne** [comment]
Ça marche, events et factures notés comme prioritaires :+1:

**Comment by webaaz** [comment]
On se fait un point rapide ce soir que je t'explique les grandes lignes

**Comment by thierrymarianne** [comment]
Ça marche ! 20h30 ?

**Comment by Wfgav** [comment]
Comment avoir accès pour prendre l'argent a la banque sans la carte visa  juste
